ftl/BUILD

```diff
@@ -43,3 +47,51 @@ py_test(
         "@mock//:mock",
     ],
 )
+
+py_test(
+    name = "node_builder_test",
+    srcs = ["node/builder_test.py"],
+    main = "node/builder_test.py",
+    deps = [
+        ":node_lib",
+        "@mock//:mock",
+    ],
+    data = [
+        ":node/testdata/base_image/config_file",
```

Comment:
you can use glob to make this easier to maintain

Reply:
done